src/Chapter06/Listing06.35C.DisablingRequiredObjectInitialization.cs

```diff
@@ -0,0 +1,69 @@
+#if NET7_0_OR_GREATER
+using System.Diagnostics.CodeAnalysis;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_34C
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            Book book = new(42) { 
+                Subtitle = "The Comprehensive, Expert Guide " +
+                    "to C# for Programmers at All Levels" };
```

Comment:
Any reason why these are capitalized?
```suggestion
                    "to C# for programmers at all levels" };
```